### DISCUSSION TITLE
Multiplex output-dir configuration

### DISCUSSION THREAD

**Discussion by jkub6** [quarto-dev/quarto-cli#4718] [Q&A]
All of the *-speaker.html files generated by revealjs multiplex don't end up in the output-dir directory specified in _quarto.yml.
Ideally (for me) they would end up right beside the client *.html presentation files. Instead, they remain beside the .qmd files.

Is this expected behavior? If so, is there a way to configure the output directory for these files?

**Comment by cscheid** [comment]
Yes, that's expected behavior, because if we put them on the output-dir, then they might get copied to the server, and that's not what you want. Those files contain have private information that would allow anyone opening *-speaker.html to control the presentation.

**Reply by jkub6** [reply]
Ah, that makes sense. It seems that the *-speaker.html files still contain relative paths that expect it to be in the output-dir folder though. For example:

```<link href="../../site_libs/revealjs/plugin/quarto-support/footer.css" rel="stylesheet">```

As this only exists in the output-dir folder, the file only seems usable when I manually move it here (unless it is possible to assemble the "site_libs" folder in two locations)

**Reply by cscheid** [reply]
Hm. That's a very good point! I'm not sure how we can square this particular circle, though. We could try to set `embed-resources: true` by default on the -speaker.html file, but that will likely generate its own set of problems.

**Reply by cscheid** [reply]
I created an issue to discuss: https://github.com/quarto-dev/quarto-cli/issues/4738
